config.js

```diff
@@ -1,6 +1,6 @@
 exports.DATABASE_URL = process.env.DATABASE_URL ||
                        global.DATABASE_URL ||
-                      'mongodb://localhost/basic-auth-demo';
+                      'mongodb://localhost/jwt-auth-demo';
 exports.PORT = process.env.PORT || 8080;
 exports.JWT_SECRET = process.env.JWT_SECRET;
```

Comment:
:-1: I worry that if we provide a default then it'll inevitably sneak into production.  Students should understand the responsibility here, and I think making them actively do something is a good prod in that direction.